charts/simple-app/values.yaml

```diff
@@ -489,6 +489,20 @@ datadog:
   metricsToScrape:
     - '"*"'
 
+  # -- (`map`) If true, then it will enable application logging to datadog.
```

Comment:
Remove this comment because this commnt implies that `.Values.scrapeLogs` should be specifically set to true, when it's really a map.. When documenting a map, document the individual settings within it, and then the `helm_docs` process generates a nic readme..

Reply:
done